python/lsst/meas/extensions/scarlet/source.py

```diff
@@ -210,15 +207,16 @@ def liteModelToHeavy(source, mExposure, blend, xy0=None, dtype=np.float32, useFl
         model = blend.observation.convolve(model, mode="real").astype(dtype)
 
     # Update xy0 with the origin of the sources box
-    xy0 = Point2I(overlap.origin[-1] + xy0.x, overlap.origin[-2] + xy0.y)
+    _xy0 = Point2I(overlap.origin[-1] + xy0.x, overlap.origin[-2] + xy0.y)
```

Comment:
I still think you should just change the function args to separate x0, y0 ints if you don't actually need the `Point2I` object. Having both `xy0` and `_xy0` in scope is bound to confuse someone eventually.

Reply:
Typically there is an `xy0` (see [`ScarletDeblendTask`](https://github.com/lsst/meas_extensions_scarlet/blob/main/python/lsst/meas/extensions/scarlet/scarletDeblendTask.py#L1025)), but it also needs to be adjusted by the location of the minimum bounding box that contains the footprint. So there has to be a second `xy0`. I guess I could just overwrite the initial `xy0` since it's never used again,.